code/modules/mining/ores_coins.dm

```diff
@@ -378,13 +382,43 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 	grind_results = list("carbon" = 4)
 
 /obj/item/coin/iron
-	name = "iron coin"
+	name = "bottle cap"
+	desc = "A humble bottle cap. This one seems to be of the Nuka Cola variety."
 	cmineral = "iron"
 	icon_state = "coin_iron_heads"
-	value = 1
+	is_non_flippable = TRUE
+	sideslist = list("heads")
+	value = 2.5
```

Comment:
for coin values, please make defines for them like COIN_VALUE_IRON, COIN_VALUE_CAP, etc. pref using cap as 1, with everything else being defined based off that
like
```
#define COIN_VALUE_CAP 1
#define COIN_VALUE_NCR 0.25 * COIN_VALUE_CAP
```

Reply:
Ok.